app/src/org/commcare/utils/CredentialShareData.kt

```diff
@@ -0,0 +1,19 @@
+package org.commcare.utils
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class CredentialShareData(
```

Comment:
this is a model class and should not be in ` org.commcare.utils` but a personalID related package. Also should be renamed to `PersonalIDCredentialShareData` for consistency.